packages/ra-ui-materialui/src/input/TranslatableInputs.tsx

```diff
@@ -101,18 +110,28 @@ export interface TranslatableInputsProps extends UseTranslatableOptions {
     className?: string;
     selector?: ReactElement;
     children: ReactNode;
+    fullWidth?: boolean;
     groupKey?: string;
     margin?: 'none' | 'normal' | 'dense';
-    variant?: 'standard' | 'outlined' | 'filled';
```

Comment:
`variant` was actually unused so I removed it completely to avoid confusion (it should be passed to the children directly)